mongo/integration/crud_prose_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package integration
+
+import (
+	"testing"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/internal/testutil/assert"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/integration/mtest"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func TestCrudProse(t *testing.T) {
+	clientOpts := options.Client().SetRetryWrites(false).SetWriteConcern(mtest.MajorityWc).
+		SetReadConcern(mtest.MajorityRc)
+	mtOpts := mtest.NewOptions().ClientOptions(clientOpts).MinServerVersion("4.0").Topologies(mtest.ReplicaSet).
+		CreateClient(false)
+	mt := mtest.New(t, mtOpts)
+	defer mt.Close()
+
+	label := "ExampleError"
+	mt.Run("InsertMany errors with RetryableWriteError label", func(mt *mtest.T) {
```

Comment:
nit: I didn't catch this in the last PR, but the test name specifies `RetryableWriteError` and the actual label is `ExampleError`. Easiest change is probably to name the test `InsertMany errors with label`?

Reply:
done